client/packages/system/src/ProgramEnrolment/ListView/ListView.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useMemo } from 'react';
+import {
+  TableProvider,
+  DataTable,
+  useColumns,
+  createTableStore,
+  NothingHere,
+  createQueryParamsStore,
+  useFormatDateTime,
+  ColumnAlign,
+  useUrlQueryParams,
+  useNavigate,
+} from '@openmsupply-client/common';
+import { useProgramEnrolment } from '../api/hooks';
+import { ProgramFragment } from '../api/operations.generated';
+import { AppBarButtons } from './AppBarButtons';
+
+type ProgramFragmentWithId = { id: string } & ProgramFragment;
+
+const ProgramListComponent: FC = () => {
+  const {
+    updateSortQuery,
+    updatePaginationQuery,
+    queryParams: { sortBy, page, first, offset },
+  } = useUrlQueryParams();
+  const { data, isError, isLoading } = useProgramEnrolment.document.list();
+  const dataWithId: ProgramFragmentWithId[] | undefined = useMemo(
+    () =>
+      data?.nodes.map(node => ({
+        id: node.name,
+        ...node,
+      })),
+    [data]
+  );
+  const pagination = { page, first, offset };
+  const { localisedDate } = useFormatDateTime();
+
+  const columns = useColumns<ProgramFragmentWithId>(
+    [
+      {
+        key: 'type',
+        label: 'label.enrolment-program',
+      },
+      {
+        key: 'programPatientId',
+        label: 'label.enrolment-patient-id',
+      },
+      {
+        key: 'enrolmentDatetime',
+        label: 'label.enrolment-datetime',
+        align: ColumnAlign.Right,
+        width: 160,
```

Comment:
the width isn't enough for the header text! 170 would do, but I would use `175` as I work in multiples of `25` 😁